demo/share/share.php

```diff
@@ -3,11 +3,10 @@
 require_once __DIR__ . '/../bootstrap.php';
 
 $loop = \React\EventLoop\Factory::create();
+$scheduler  = new \Rx\Scheduler\EventLoopScheduler($loop);
 
 //Without Share
-$interval = new \Rx\React\Interval(1000, $loop);
-
-$source = $interval
+$source = (new \Rx\Observable\IntervalObservable(1000, $scheduler))
```

Comment:
I've been trying to use the static methods in the demos (I think that they're easier to read).
Can you change these to `Rx\Observable::interval(1000, $loop)` instead of declaring a new class?